hack/make-rules/kustomize_to_chart.sh

```diff
@@ -261,6 +261,8 @@ EOF
     append_note $yurt_manager_templatefile
 
     kubectl kustomize ${output_default_dir} >> $yurt_manager_templatefile
+
+    sed "s/namespace: kube-system/namespace: {{ .Release.Namespace | quote }}/g" $yurt_manager_templatefile > tmpfile && mv tmpfile $yurt_manager_templatefile
```

Comment:
don't need to modify `yurt-manager-auto-generated.yaml` file. because namespace in `yurt-manager-auto-generated.yaml` file will be overwrite in yurt-manager with running namespace, the detail code is here: 
https://github.com/openyurtio/openyurt/blob/master/pkg/webhook/util/configuration/configuration.go#L68
https://github.com/openyurtio/openyurt/blob/master/pkg/webhook/util/configuration/configuration.go#L93

Reply:
Thank you for your explanation. It has been corrected.